### DISCUSSION TITLE
How are packages handeld, that can be compiled with different libraries/methods

### DISCUSSION THREAD

**Discussion by leoheitmannruiz** [Homebrew/discussions#4965] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
not relevant
```

### Output of `brew doctor`

```shell
not relevant
```

### Description of issue

Does Homebrew have a solution for packages, which can be compiled with different libraries/methods?

For example, in the case of LibEtPan and[this](https://github.com/Homebrew/homebrew-core/issues/54182) issue.

It seems LibEtpan can be compiled using either GnuTLS or OpenSSL (or both?) and either using GNU Autotools or Xcode.

Claws Mail needs (it seems) LibEtPan compiled using Autotools and GnuTLS, others (I guess) need compilation using Xcode.

**Comment by SMillerDev** [comment]
Compile it twice, or just pick the most popular configuration.
